apps/chat_server/lib/chat_server/schemas/message.ex

```diff
@@ -10,6 +10,8 @@ defmodule ChatServer.Schema.Message do
     belongs_to :room, ChatServer.Schema.Room
     belongs_to :user, ChatServer.Schema.User
 
+    many_to_many :users, ChatServer.Schema.User, join_through: "starred_messages"
```

Comment:
Is there a more descriptive name we could use? I wouldn't necessarily make the mental connection between `%Message{users: ...}` pointing to "Users who have starred this message".

What if we used something like `stargazers`? Or `starred_users` or `star_users`? Spitballing here, additional ideas welcome!